src/serverdlg.cpp

```diff
@@ -451,6 +474,11 @@ void CServerDlg::OnCentServAddrTypeActivated ( int iTypeIdx )
     UpdateGUIDependencies();
 }
 
+void CServerDlg::OnNewRecordingClicked()
```

Comment:
These "mini function bodies" I usually implement right in the header, see, e.g., here: https://github.com/corrados/jamulus/blob/master/src/connectdlg.h#L111

Reply:
Good point - I've been doing that, don't know why this one slipped through.